### PR TITLE
feat: move out buttons from settings

### PR DESCRIPTION
**Pull request by henriettemoe**

### CODE REVIEW

File: language/nn.json [hunk 1]
```diff
@@ -74,12 +74,12 @@
           "default": "Ingen gyldige ord funne. Kontroller orda dine og prøv igjen."
         },
         {
-          "label": "Skjermlesartekst for \"Opne innstillingar\" knapp",
-          "default": "Opne innstillingar"
+          "label": "Etikett for \"Fyll inn\" svarmodus",
+          "default": "Fyll in"
```
Comment (boyum) [inline language/nn.json:78]:
```suggestion
          "default": "Fyll inn"
```

File: language/nb.json [hunk 2]
```diff
@@ -74,12 +74,12 @@
           "default": "Ingen gyldige ord funnet. Kontroller ordene dine og prøv igjen."
         },
         {
-          "label": "Skjermlesertekst for \"Åpne innstillinger\" knapp",
-          "default": "Åpne innstillinger"
+          "label": "Etikett for \"Fyll inn\" svarmodus",
+          "default": "Fyll in"
```
Comment (boyum) [inline language/nb.json:78]:
```suggestion
          "default": "Fyll inn"
```

File: semantics.json [hunk 3]
```diff
@@ -142,7 +142,7 @@
       {
         "label": "Language mode label",
         "name": "languageModeLabel",
-        "default": "Change language mode",
+        "default": "Swap language",
```
Comment (boyum) [inline semantics.json:145]: Swap language or swap languages?

```suggestion
        "default": "Swap languages",
```

Comment (henriettemoe) [reply semantics.json:145]: Good question 🤔 What do you think?

File: src/components/Toolbar/Toolbar.tsx [hunk 4]
```diff
@@ -1,27 +1,70 @@
...
 }) => {
   const { t } = useTranslation();
 
+  const enableTools = enableAnswerMode || enableLanguageMode;
+
+  const getSelectedAnswerMode = (): string => {
+    if (activeAnswerMode === AnswerModeType.FillIn) {
+      return 'h5p-vocabulary-drill-fill-in';
+    }
+    return 'h5p-vocabulary-drill-drag-text';
```
Comment (boyum) [inline src/components/Toolbar/Toolbar.tsx:30]: Another way to do this is by matching:

```suggestion
    const classes = {
      [AnswerModeType.FillIn]: 'h5p-vocabulary-drill-fill-in',
      [AnswerModeType.DragText]: 'h5p-vocabulary-drill-drag-text',
    };

    return classes[activeAnswerMode];
```

Also, by adding a parameter to the function, it's quite easy to move it out of the component, and by extension out of the render function. Re-creating a function is in general kind of wasteful, even though in this case it wouldn't matter much.

File: semantics.json (hunk 3 above)
Comment (henriettemoe) [reply semantics.json]: Used languages

Comment (boyum) [reply semantics.json]: I think maybe plural as we're swapping to things

File: src/components/Toolbar/Toolbar.tsx [hunk 5]
```diff
@@ -1,27 +1,68 @@
...
 type ToolbarProps = {
   title: string;
-  enableSettings: boolean;
-  showSettings: boolean;
-  toggleShowSettings: () => void;
+  activeAnswerMode: AnswerModeType;
+  enableAnswerMode: boolean;
+  enableLanguageMode: boolean;
+  handleAnswerModeChange: () => void;
+  handleLanguageModeChange: () => void;
```
Comment (boyum) [inline src/components/Toolbar/Toolbar.tsx:11]: We haven't talked much about this and I definitely haven't followed my own rules here, but there's a convention to name event function props `onX`. The function itself is often called `handleX`.

```tsx
const Parent = () => {
  const handleUpdate = (data: Data) => { ... };

  return <Child onUpdate={handleUpdate} />;
};
```

We don't have to fix this now (or ever), but we might want to follow this convention in the future.

Comment (henriettemoe) [reply src/components/Toolbar/Toolbar.tsx]: Added

File: src/index.scss [hunk 6]
```diff
@@ -1,129 +1,142 @@
...
+    padding: 0.75em 2.75em 0.75em 2.7em;
 
-    &:before {
-      font-family: "H5PFontIcons";
-      content: "\e916";
-      line-height: 1;
+    // Remove default arrow for lower IE versions
+    &::-ms-expand {
+      display: none;
     }
```
Comment (boyum) [inline src/index.scss:98]: hello, ie10 👀

Comment (henriettemoe) [reply src/index.scss:98]: :)

File: src/components/Toolbar/Toolbar.tsx (hunk 4 above)
Comment (henriettemoe) [reply src/components/Toolbar/Toolbar.tsx]: Better now?

**Review by boyum — APPROVED** [review]

**Pull request merged**